site-src/concepts/api-overview.md

```diff
@@ -75,35 +76,51 @@ Status object.
 A Gateway MAY contain one or more *Route references which serve to direct
 traffic for a subset of traffic to a specific service.*
 
-### {HTTP,TCP,Foo}Route
+### Route objects
 
 Route objects define protocol-specific rules for mapping requests from a Gateway
 to Kubernetes Services.
 
-`HTTPRoute` and `TCPRoute` are currently the only defined Route objects.
-Additional protocol-specific Route objects may be added in the future.
+As of v1alpha2, the API defines four Route types, although it's
+expected that in the future implementations may create their own custom Route
+types.
 
-### BackendPolicy
+#### HTTPRoute
+HTTPRoute is for multiplexing HTTP or terminated HTTPS connections. It's intended
+for use in cases where you want to inspect the HTTP stream and use HTTP-level data
+for either routing or modification, for example using HTTP Headers for routing, or
+modifying them in-flight.
 
-BackendPolicy provides a way to configure connections between a Gateway and a
-backend. For the purpose of this API, a backend is any resource that a route can
-forward traffic to. A common example of a backend is a Service. Configuration at
-this level is currently limited to TLS, but will expand in the future to support
-more advanced policies such as health checking.
+#### TLSRoute
+TLSRoute is for multiplexing TLS connections, discriminated via SNI. It's intended
+for where you want to use the SNI as the main routing method, and are not interested
+in properties of the underlying connection (such as whether or not it is HTTP).
+
+#### TCPRoute and UDPRoute
+TCPRoute (and UDPRoute) are intended for use as a mapping between a single port or
+set of ports and a single backend. In this case, there is no discriminator you can
+use to choose different backends on the same port, so each TCPRoute really needs a
+different port on the listener (in general, anyway). You can still terminate TLS,
+and passthrough will work since the stream is passed through unchanged. 
```

Comment:
```suggestion
different port on the listener (in general, anyway). You can terminate TLS in
which case the unencrypted byte stream is passed through to the backend.
You can chose to not terminate TLS, in which case the encrypted byte stream
is passed through to the backend. 
```
More explicit here because I expect major confusion here.